incompatibility/incompatibility-report/incomp-scanning.go

```diff
@@ -39,10 +41,32 @@ func CreateIncompReport(env *plugins.Environment, reportType Report) {
 
 // Scan for incompatibilities in an OpenAPI document
 func ScanIncompatibilities(document *openapiv3.Document) *IncompatibilityReport {
-	var incompatibilities []*Incompatibility
 
-	if document.Servers != nil {
-		incompatibilities = append(incompatibilities, &Incompatibility{TokenPath: []string{"servers"}, Classification: "SERVERS"})
+	paths, err := knownIncompatibilityPaths()
+	if err != nil {
+		return &IncompatibilityReport{}
```

Comment:
I would expect you to return something like `nil, fmt.Errorf("failed to scan for incompatibilities: %s", err` here. To communicate issues up better. Same with the return after the compile call.